source/change-streams/change-streams.rst

```diff
@@ -534,6 +536,29 @@ The `CursorKilled` or `Interrupted` error implies implies some other actor kille
 The `CappedPositionLost` error implies falling off of the back of the oplog,
 so resuming is impossible.
 
+-----------------------------------------------------------------------------------------
+Why do we need to send a default ``startAtOperationTime`` in the ``$changeStream`` stage?
+-----------------------------------------------------------------------------------------
+
+``startAtOperationTime`` allows a user to create a change stream that does not return
```

Comment:
how about: "allows a user to create a resumable change stream even when a result (and corresponding resultToken) is not available until a later point in time."